api/awsauth/awsauth/config.py

```diff
@@ -0,0 +1,28 @@
+import pydantic
+
+
+class EnvConstants(pydantic.BaseSettings):
+    """Environment constants.
+
+    Constants should be set in .env file with keys matching variable names
+    at root of project.
+    """
+
+    API_KEY_TABLE_NAME: str
+
+    SENTRY_DSN: str
+
+    EMAILS_ENABLED: bool
+
+    class Config:
+        env_file = ".env"
+        env_file_encoding = "utf-8"
+
+
+class Config:
+
+    Constants: EnvConstants = None
```

Comment:
Is there a case where we'd want to create a Config instance? If not, how about putting `constants: EnvConstants = None` at the module level?

Reply:
definitely fine with that!

Reply:
Ah so the reason that I actually don't do that is so that we can initialize the environment at runtime rather than import. Because otherwise, I believe that references to constants will get set at import time? though I'm now questioning if that's actually right

Reply:
I'm imagining the module level variable is None until the init function is called, so the env is not read during import.

Reply:
Right - you would just have to be careful to not import `constants` directly (i.e. `from awsauth.config import constants`) or constants will be none.  However:
```
from awsauth import config
config.constants
```
would work alright